tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -194,6 +194,10 @@ sub run_test {
     #FOR S390X LPAR
     if (check_var('ARCH', 's390x')) {
         virt_utils::lpar_cmd("$test_cmd");
+        if (get_var("VIRT_PRJ4_GUEST_UPGRADE")) {
+            upload_asset("/tmp/guest_upgrade_test.log", 1, 1);
```

Comment:
How about s390x_guest_upgrade_test ?

Reply:
Because of there was just only called test_full_guest_upgrade.sh shell script to trigger s390x guest upgrade test, was included one time ctcs2, means that missed required test log guest_upgrade_test.log here.
Thus, add this missed guest_upgrade_test.log for osd s390x guest upgrade test.